commit ecc7b5f332674ee57ad11a0d6bb58a252549c762
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Tue Sep 22 13:03:23 2009 +0000

    Don't search LIBGL_DRIVERS_PATH, stick to LIBVA_DRIVERS_PATH only or the default path.

diff --git a/va/va.c b/va/va.c
index c8e036e..ada70ce 100644
--- a/va/va.c
+++ b/va/va.c
@@ -127,10 +127,6 @@ static VAStatus va_openDriver(VADisplay dpy, char *driver_name)
     {
         /* don't allow setuid apps to use LIBVA_DRIVERS_PATH */
         search_path = getenv("LIBVA_DRIVERS_PATH");
-        if (!search_path)
-        {
-            search_path = getenv("LIBGL_DRIVERS_PATH");
-        }
     }
     if (!search_path)
     {
